gammapy/region/reflected.py

```diff
@@ -0,0 +1,68 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from . import RegionList, PixCircleRegion
+
+__all__ = [
+    'find_reflected_regions',
+]
+
+
+def find_reflected_regions(region, center, exclusion_mask,
+                           angle_increment=0.1, min_distance=0):
+    """Find reflected regions.
+
+    Converts to pixel coordinates internally
+
+    Parameters
+    ----------
+    region : `~gammapy.region.Region`
+        Region
+    center : `~astropy.coordinates.SkyCoord`
+        Rotation point
+    exclusion_mask : `~gammapy.region.ExclusionMask`
+        Exlusion mask
+    angle_increment : float
+        Rotation angle for each step
+    min_dinstance : float
+        Minimal distance from input region
+
+    Returns
+    -------
+    regions : `~gammapy.region.RegionList`
+        Reflected regions list
+    """
+    
+    reflected_regions_pix = RegionList()
+    wcs = exclusion_mask.wcs
+    pix_region = region.to_pixel(wcs)
+    val = center.to_pixel(wcs, origin=1)
+    pix_center = (float(val[0]), float(val[1]))
+    offset = pix_region.offset(pix_center)
+    angle = pix_region.angle(pix_center)    
+    min_ang = 2 * pix_region.radius / offset
+    max_angle = angle + 2 * np.pi - min_ang - min_distance
+
+    curr_angle = angle + min_ang + min_distance
+    found_region = False
+    while curr_angle < max_angle:
+        test_pos = _compute_xy(pix_center, offset, curr_angle)
+        test_reg = PixCircleRegion(test_pos, pix_region.radius)
+        if not test_reg.is_inside_exclusion(exclusion_mask):
+            reflected_regions_pix.append(test_reg)
+            curr_angle = curr_angle + min_ang
+        else:
+            curr_angle = curr_angle + angle_increment
+     
+    reflected_regions = reflected_regions_pix.to_sky(wcs)
+    return reflected_regions
+
+def _compute_xy(pix_center, offset, angle):
```

Comment:
Astropy coordinates is also for 2D and 3D cartesian coordinates.
But if you're not familiar with `astropy.coordinates` (I only know it a little) it takes time to figure out how to to something like this.
So maybe just put a comment for now?
```python
# TODO: replace by calculation using `astropy.coordinates`
```